code/modules/mob/living/carbon/human/species/station.dm

```diff
@@ -93,7 +93,7 @@
 
 /datum/action/innate/tail_lash/Activate()
 	var/mob/living/carbon/human/user = owner
-	if(!user.restrained() || !user.buckled)
+	if(user.restrained() || user.buckled)
```

Comment:
If you want a restrained Unathi that isn't being pulled by someone (i.e. has cuffs on but can still move) to be able to do this, you just have to amend this line so it's 

`if((user.pulledby && user.restrained()) || user.buckled)`

Reply:
Do this please ^